usr/src/lib/brand/lx/zone/lx_boot_zone_busybox.ksh

```diff
@@ -36,11 +36,16 @@ depend() {
     keyword nojail noprefix novserver
 }
 start() {
+EOF
+# Only alter resolve.conf if we're getting info from zonecfg(1M).
```

Comment:
```suggestion
# Only alter resolv.conf if we're getting info from zonecfg(1M).
```

Reply:
You have this flaw in your gate too, and I'll fix it here (again).